packages/manager/modules/pci/src/projects/project/streams/stream/stream.html

```diff
@@ -0,0 +1,95 @@
+<oui-page-header data-heading="{{ $ctrl.stream.name }}">
```

Comment:
```suggestion
<oui-page-header data-heading="{{:: $ctrl.stream.name }}">
```
wherever it is possible